### ISSUE TITLE
whether the organizers can release the ground truth of the validation dataset?

### ISSUE THREAD

**Issue by tianmaliuixing** [paip-2019/challenge#45]
Dear PAIP organizers,
The validation dataset competition has ended. It can be seen from the leaderboard that there are several cases that almost all teams have not predicted, and the dataset also shows that these cases are very different from the distribution of training datasets.  I wonder whether the organizers can release the ground truth of the validation dataset, so that we have the opportunity to continue to explore the problem.
Thanks,

**Comment by wisepaip** [comment]
Thank you for the suggestion. 
We plan to open the validation ground truth after the test deadline. 
We will let you know when the data is updated.

Thank you.

**Comment by yangsenwxy** [comment]
I wonder when the organizers can release the ground truth of the test dataset.

**Comment by wisepaip** [comment]
Hello, the validation ground truth has been released on our website. 
Since we re-open the submission on the challenge website, we don’t have a plan to open the ground truth for the test set right now.
Thank you.

**Comment by yangsenwxy** [comment]
Hi,can you share the validation ground truth link?

**Comment by wisepaip** [comment]
@yangsenwxy You could find the ground truth of validation at PAIP 2019 challenge data download website. 
Quick remind you that you could access PAIP 2019 and PAIP 2020 datasets with the same password. However, your previous password would be renewed when you got the recent credentials email.

**Comment by yangsenwxy** [comment]
@wisepaip I have sent an email, can you reply me？
